src/Handler/CurlMultiHandler.php

```diff
@@ -214,6 +242,8 @@ private function timeToNext(): int
             }
         }
 
-        return \max(0, $nextTime - $currentTime) * 1000000;
+        $timeToNext = (int) \max(0, $nextTime - $currentTime);
+
+        return $timeToNext * 1000000;
```

Comment:
Shall we move this to a one line with a paranthesis?

Reply:
Fixed in 44148c15357f5601487877f3d8bf8a97a8c4b1af